website_blog_no_background_image/models/blog_blog.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2015 Therp B.V, Giovanni Francesco Capalbo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import fields, models
+
+
+class WebsiteBlog(models.Model):
+
+    _inherit = 'blog.blog'
+
+    def _get_image_options(self):
+        return [('no_image', "No header image"),
+                ('small_image', "Show small header image"),
+                ('big_image', "Show big header image")
+                ]
+
+    background_image_show = fields.Selection(
+        string="Default blog-wide setting for background image",
+        selection=lambda self: self._get_image_options(),
```

Comment:
Why don't use a static list now that it's easy to be extended with selection_add? Your current method doesn't translate available options.